integrations/segment/gitbook-manifest.yaml

```diff
@@ -33,6 +34,14 @@ summary: |
 categories:
     - analytics
 configurations:
+    site:
+        properties:
+            write_key:
+                type: string
+                title: Write Key
+                description: Key provided by Segment to identify the data source.
+        required:
+            - write_key
```

Comment:
Does it mean that users will have to reconfigure or can we run a migration?

Reply:
yeah we will have to run a migration to migrate space level installation to their sites